vendor/github.com/pilosa/go-pilosa/gopilosa_pbuf/public.pb.go

```diff
@@ -544,42 +544,18 @@ func (m *ImportValueRequest) GetValues() []int64 {
 }
 
 func init() {
-	if t := proto.MessageType("internal.Bitmap"); t == nil {
-		proto.RegisterType((*Bitmap)(nil), "internal.Bitmap")
-	}
-	if t := proto.MessageType("internal.Pair"); t == nil {
-		proto.RegisterType((*Pair)(nil), "internal.Pair")
-	}
-	if t := proto.MessageType("internal.ValCount"); t == nil {
-		proto.RegisterType((*ValCount)(nil), "internal.ValCount")
-	}
-	if t := proto.MessageType("internal.Bit"); t == nil {
-		proto.RegisterType((*Bit)(nil), "internal.Bit")
-	}
-	if t := proto.MessageType("internal.ColumnAttrSet"); t == nil {
-		proto.RegisterType((*ColumnAttrSet)(nil), "internal.ColumnAttrSet")
```

Comment:
good to keep these extra ifs. it avoids printing warning about already registered type, on start.

Reply:
Vendor files should not be modified manually

Reply:
We'll have to find a better way for these autogenerated files, but so far I modified them adding extra ifs to avoid nasty warnings (after integrating pilosalib driver - both try to register the same types in grpc)

Reply:
Actually, I didn't have protobuf errors on startup anymore. Some dependency must be updated.

Reply:
@ajnavarro - do you have pilosa running? Right now, if pilosa is not running we don't register old driver. The warnings appeared when we registered both drivers, because `pilosa` and `go-pilosa` register in protobuf the same types.

@erizocosmico - I totally agree, but it was just ad-hoc _fix_ to suppress ugly warnings. It was temporary, the quickest fix.

Moreover based on comments in protobufs:
https://github.com/golang/protobuf/blob/master/proto/properties.go#L481
Some day _duplicate proto type register_ may panic.